### PR TITLE
Sample to create a pipeline from a component

### PR DESCRIPTION
**Pull request by joeliedtke**
Adds sample code to create a pipeline from an AI Hub component by URL or by file path.

### CODE REVIEW

File: ai-hub/kubeflow-pipeline/create-pipeline-from-component-file.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+# [START load_component_file]
+scikit_learn_train = comp.load_component_from_file('[PATH_TO_COMPONENT_ZIP_FILE]')
+# [END load_component_file]
+
+# [START create_pipeline]
+# Use the @dsl.pipeline decorator to add a name and description to your pipeline definition.
+@dsl.pipeline(
+    name='Scikit-learn Trainer',
+    description='Trains a Scikit-learn model')
```
Comment (gogasca) [inline ai-hub/kubeflow-pipeline/create-pipeline-from-component-file.py]: nip: Description: scikit-learn vs Scikit-learn

File: ai-hub/kubeflow-pipeline/create-pipeline-from-component-url.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
+#!/bin/python
+# Copyright 2017 Google Inc. All Rights Reserved.
```
Comment (gogasca) [inline ai-hub/kubeflow-pipeline/create-pipeline-from-component-url.py]: nit: Copyright year to: 2019

File: ai-hub/kubeflow-pipeline/create-pipeline-from-component-file.py [hunk 3]
```diff
@@ -0,0 +1,46 @@
+#!/bin/python
+# Copyright 2017 Google Inc. All Rights Reserved.
```
Comment (gogasca) [inline ai-hub/kubeflow-pipeline/create-pipeline-from-component-file.py]: nit: Copyright year to 2019

**Comment by andrewferlitsch** [comment]
@gguuss Is there an update to this PR?

**Pull request closed without merging**